### DISCUSSION TITLE
Multiple Output

### DISCUSSION THREAD

**Discussion by dezenn** [lululxvi/deepxde#1757] [Q&A]
My code is based on the demo 'antiderivative_aligned_pideeponet.py'. I use my own data and it works fine. Now I want to use two target functions and I see that DeepONetCartesianProd has the argument 'num_outputs' but I wonder what data.module I should use? When I use TripleCartesianProd I run into shape errors.

**Comment by lululxvi** [comment]
Not exactly what you need, but see https://github.com/lululxvi/deepxde/blob/master/examples/operator/stokes_aligned_pideeponet.py
